network_importer/adapters/netbox_api/models.py

```diff
@@ -33,6 +33,35 @@
 
 LOGGER = logging.getLogger("network-importer")
 
+from diffsync.enum import DiffSyncModelFlags
+
+def _is_tag_present(obj):
+    """Find if tag is present for a given object."""
+    
+    tags = list()
+    for tag in config.SETTINGS.main.model_flag_tags:
+        if tag in obj['tags']:
+            LOGGER.debug(
+                "Tag (%s) found identified for object %s. Marked for diffsync flag assigment.",
+                tag,
+                obj,
+            )
+            return True
+    return False
+
+def apply_model_flag(item, obj):
```

Comment:
`item`, `obj` are not super clear names. `item` is the DiffSync model instance and `obj` is the corresponding PyNautobot record, correct?

Reply:
Agree. Names updated.